app/src/main/java/com/example/android/lifecycles/step5_solution/Fragment_step5.java

```diff
@@ -31,6 +34,8 @@
 import com.example.android.codelabs.lifecycle.R;
 import com.example.android.lifecycles.step5.SeekBarViewModel;
 
+import static java.util.Objects.requireNonNull;
```

Comment:
can remove this if using `requireActivity()`

Reply:
done